Examples/test-suite/javascript/abstract_access_runme.js

```diff
@@ -1,6 +1,8 @@
-var abstract_access = require("abstract_access");
+var abstract_access = require("./abstract_access");
 
 var d = new abstract_access.D()
 if (d.do_x() != 1) {
    throw "Error";
 }
+
+process.exit(0);
```

Comment:
What is the reason for explicit exit call? AFAIK if successful, the process always exits with 0.

Reply:
That's my assumption as well, but on electron the process would not exit unless this call. I lack knowledge of electron to know if that's right behavior though, but that felt like an acceptable midway step until I could get a proper review done.